files/bootstrap.sh

```diff
@@ -385,13 +380,17 @@ sed -i s,MASTER_ENDPOINT,$APISERVER_ENDPOINT,g /var/lib/kubelet/kubeconfig
 sed -i s,AWS_REGION,$AWS_DEFAULT_REGION,g /var/lib/kubelet/kubeconfig
 ### kubelet.service configuration
 
-if [[ "${IP_FAMILY}" == "ipv6" ]]; then
-      DNS_CLUSTER_IP=$(awk -F/ '{print $1}' <<< $SERVICE_IPV6_CIDR)a
-fi
-
 MAC=$(get_meta_data 'latest/meta-data/network/interfaces/macs/' | head -n 1 | sed 's/\/$//')
 
 if [[ -z "${DNS_CLUSTER_IP}" ]]; then
+  if [[ "${IP_FAMILY}" == "ipv6" ]]; then
+    if [[ -z "${SERVICE_IPV6_CIDR}" ]]; then
+      echo "Service Ipv6 Cidr must be provided when ip-family is specified as IPV6"
```

Comment:
Can we update this error message saying "either service ipv6 cidr or cluster-dns-ip need to be present"?